blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/JSONMetricsContainer.java

```diff
@@ -64,14 +65,18 @@ public JSONMetricsContainer(String tenantId, List<JSONMetric> metrics) {
         delayedMetrics = new ArrayList<Metric>();
         for (JSONMetric jsonMetric : jsonMetrics) {
             Locator locator;
+            boolean isValid;
```

Comment:
No because of the scoped extension of the metric.  See the static subclass of JsonMetric called ScopedJSONMetric in the same file (/JSONMetricsContainer.java).